### PR TITLE
Translations/fr fr

### PR DESCRIPTION
**Pull request by Tagada85**
Well, I have to start somewhere 😄
I created a new folder in `11ty/definitions` called `fr-FR` to store the translations.
So far, I tried to translate the *ok-sign* and *gaslighting* definitions.

Let me know if there is anything to modify about the structure or anything else 😉

### CODE REVIEW

**Comment by Tagada85** [comment]
Fixed some things following @hteumeuleu remarks!

**Comment by tatianamac** [comment]
@hteumeuleu Do you have time to review @Tagada85 's revised notes? Merci bien !

**Comment by tatianamac** [comment]
Hello @hteumeuleu and @Tagada85 ! I recognise these are weird times. I hope you're all able to care for yourselves. 💙

I'm hoping to clean through these older pull requests. How are you feeling about these translations? 

No pressure either way; I'm mostly just hoping for your recommendation on what their status is so I can figure out if they're presentable as they are or need more work.

**Comment by Tagada85** [comment]
Fixed a couple of typos I found reviewing the definitions. All good for me 😄

**Comment by Tagada85** [comment]
I added the white fragility and cisgender translations.

I did a rebase thingy to make my up to date... I think I didn't break anything but if someone could double check 😬 

@hteumeuleu If you have the time/courage to review this, I would especially like to have your opinion on the Impact part of the White Fragility definition. I've had some issues to make it clear enough and would like to know if you think it could be improved. Thanks  🙏

**Comment by tatianamac** [comment]
@Tagada85 Thank you for adding those! 

For future definitions, do you mind opening a separate PR? I'd like to merge in this PR so keep them a bit cleaner in the future. (We can finish out `white fragility` and `cisgender` under this PR, no problem) Merci bien ! 💙

File: 11ty/definitions/fr-FR/capacitisme.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+---
+title: Capacitisme
+slug: ableism
```
Comment (tatianamac) [inline 11ty/definitions/fr-FR/capacitisme.md:3]: `capacitisme` for the slug too

**Comment by tatianamac** [comment]
@Tagada85 I added an ongoing issue (#125) to help us track French translations. This way, anytime you or anyone who wants to contribute can communicate which words they'll define there.

For each word/small set of words, we can open individual PRs, which will keep things a bit tidier and easier to maintain. 💙

**Comment by Tagada85** [comment]
Sure! Noted for the future 😃

**Comment by tatianamac** [comment]
@hteumeuleu Do you still have time/energy/desire to review @Tagada85 's two additional translations? No worries if not, I can find someone.

**Comment by hteumeuleu** [comment]
I will give this a look over the week end. Thanks for your patience!

**Comment by tatianamac** [comment]
Thank you both for these definitions! I will work on getting these linked to their English definitions.

Is it okay with you if I thank you on the Self-Defined Twitter account?

**Pull request merged**